src/codegen/sys/cargo_toml.rs

```diff
@@ -93,6 +93,15 @@ fn fill_empty(root: &mut Table, env: &Env, crate_name: &str) {
         set_string(dep, "package", ext_package);
         set_string(dep, "git", repo_url);
     }
+
+    {
+        let features = upsert_table(root, "features");
```

Comment:
It doesn't seem to automatically add the `dox` feature with the corresponding dependencies if it's missing.

I tried removing the dox feature from pangocairo-sys and regenerated, but it wasn't added back.

Reply:
Moving this to `fill_in` from `fill_empty` seems to do the right thing as @bilelmoussaoui suggested

```diff
diff --git a/gdk-pixbuf/sys/Cargo.toml b/gdk-pixbuf/sys/Cargo.toml
index 3508c36be21..bca362137e7 100644
--- a/gdk-pixbuf/sys/Cargo.toml
+++ b/gdk-pixbuf/sys/Cargo.toml
@@ -21,7 +21,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "gobject/dox", "gio/dox"]
 v2_40 = []
 v2_42 = ["v2_40"]
 
diff --git a/gio/sys/Cargo.toml b/gio/sys/Cargo.toml
index 29d349db48c..7dd6edcbe50 100644
--- a/gio/sys/Cargo.toml
+++ b/gio/sys/Cargo.toml
@@ -20,7 +20,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "gobject/dox"]
 v2_58 = []
 v2_60 = ["v2_58"]
 v2_62 = ["v2_60"]
diff --git a/glib/gobject-sys/Cargo.toml b/glib/gobject-sys/Cargo.toml
index ad64a68f693..3da746af848 100644
--- a/glib/gobject-sys/Cargo.toml
+++ b/glib/gobject-sys/Cargo.toml
@@ -13,7 +13,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox"]
 v2_58 = []
 v2_62 = ["v2_58"]
 v2_66 = ["v2_62"]
diff --git a/graphene/sys/Cargo.toml b/graphene/sys/Cargo.toml
index ba0d74cdea3..a3aac433317 100644
--- a/graphene/sys/Cargo.toml
+++ b/graphene/sys/Cargo.toml
@@ -14,7 +14,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox"]
 
 [lib]
 name = "graphene_sys"
diff --git a/pango/sys/Cargo.toml b/pango/sys/Cargo.toml
index 7cb0d098ec4..81df8950383 100644
--- a/pango/sys/Cargo.toml
+++ b/pango/sys/Cargo.toml
@@ -17,7 +17,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "gobject/dox"]
 v1_42 = []
 v1_44 = ["v1_42"]
 v1_46 = ["v1_44"]
diff --git a/pangocairo/sys/Cargo.toml b/pangocairo/sys/Cargo.toml
index 176d5ba174b..58979fc4cf7 100644
--- a/pangocairo/sys/Cargo.toml
+++ b/pangocairo/sys/Cargo.toml
@@ -21,7 +21,7 @@ shell-words = "1.0.0"
 tempfile = "3"
 
 [features]
-dox = []
+dox = ["glib/dox", "pango/dox", "cairo/dox"]
 
 [lib]
 name = "pango_cairo_sys"
```